demystifying_large_language_models/demystifying_large_language_models.md

```diff
@@ -0,0 +1,630 @@
+
+## LLM Costs and Risks
+
+Costs
+---
+
+Over time, with advances in the speed and size of computer processors (especially Graphical Processing Units, or GPUs), faster and more efficient data storage, and competition among cloud providers, computing has become less expensive.  That means that tasks can be performed today that would have been unrealistic (in terms of time and expense) to perform a few years ago.  That's true across computing, and it's certainly true in artificial intelligence.  But "less expensive" doesn't mean free.  Large language models are incredibly resource intensive, in terms of the amount of input required, the amount of computing required, and the energy consumed in building and running the models. 
+
+
+Risks
+---
+
+Additionally, there are important risks to consider.  We're going to borrow a framework from Weidinger et al.  In their superb and approachable ["Ethical and social risks of harm from Language Models,"](https://arxiv.org/abs/2112.04359) Weidinger and her co-authors enumerate six domains of risks, harms, and costs associated with large language models.  We summarize these risks below (and note, we have avoided the temptation to ask a large language model to summarize the text for us):
+
+1. Discrimination, exclusion, and toxicity
+-----
+
+Where can one obtain huge amounts of text, generated by humans, for free?  The internet!  Language models and their abilities to predict text have grown in part because of better, faster, and cheaper computers, but also in part because of the massive proliferation of human-generated text that can be analyzed.  Unfortunately, this set of language is biased in terms of who generates it and the opinions they share.  Toxic language, stereotypes, and bias abound, and these have already been documented in text generated by LLMs.  
+
+2. Information Hazards
+------
+
+Because the training data given to LLMs is so vast, there are cases (some involving litigation) of private information such as home addresses being leaked by LLMs.  Additionally, LLMs might correctly infer and disclose sensitive information while amassing and summarizing data (for example, accurately disclosing a serious disease diagnosis to an unprepared user asking about their symptoms).
+
+While the authors do not explicitly mention this, another information hazard we want to point out is the risk related to input text.  Users can disclose private or regulated information to an LLM via prompts.  One of the ways that LLMs acquire additional language samples is by adding the prompts that users supply to their vast collection of language examples.  Supplying private information (about human subjects, about companies, and so on) when asking LLMs to generate text could constitute an impermissible breach of privacy or violation of intellectual property law or other regulation.
```

Comment:
```suggestion

<div class = "important">
<b style="color: rgb(var(--color-highlight));">Important note</b><br>

While the authors do not explicitly mention this, another hazard we want to point out is the risk related to input text.  Users can disclose private or regulated information to an LLM via prompts.  One of the ways that LLMs acquire additional language samples is by adding the prompts that users supply to their vast collection of language examples.  Supplying private information (about human subjects, about companies, and so on) when asking LLMs to generate text could constitute an impermissible breach of privacy or violation of intellectual property law or other regulation.  This risk isn't technically an information hazard, in the traditional sense, but it is information-adjacent.  

I</div>
```

Reply:
@franzenr what do you think?

Reply:
I think this is important enough to be a warning highlight box, not an "important note" box. I also think its important enough that the language should be more explicit about "HEY DO NOT DO THIS" as in my suggestion. 

If you prefer the rest of the language of your version, that's fine, but I think this is crucial enough that we should be highlighting the actionable takeaway from this much more explicitly. (This seems significantly more important to me to highlight than, e.g., not giving private info to regex101, which of course is still important, but has much lower risk in terms of info leakage.) 

If you don't think this is the place for that to be highlighted, I'm also okay with that -- just suggest another place to make this a warning. I'm not sure I'm comfortable passing this through QA without a warning notice somewhere that says in plain language what not to do. 

@pm0kjp